ts/features/profile/store/reducers/profile.ts

```diff
@@ -24,4 +28,47 @@ const profileReducer = (
   return state;
 };
 
+export const profileSelector = (state: GlobalState): ProfileState =>
+  state.features.profile;
+
+export const profileFullNameSelector = createSelector(
+  profileSelector,
+  (profile: ProfileState): Option<string> =>
+    pot.getOrElse(
+      pot.map(profile, p =>
+        fromNullable(p.name).chain(n =>
+          fromNullable(p.family_name).map(f => `${n} ${f}`)
+        )
+      ),
+      none
+    )
```

Comment:
Since both the `name` and the `family_name` are required for an `InitializedProfile` what do you think about refactoring in this way?
```suggestion
  (profile: ProfileState): string | undefined =>
    pot.getOrElse(
      pot.map(profile, p => p.fiscal_code),
      undefined
    )
```
I don't see advantages to using an `Option` instead of the value, but feel free to discard the suggestion.

Reply:
maybe you mean a selector like this
```typescript 
export const profileFullNameSelector = createSelector(
  profileSelector,
  (profile: ProfileState): string | undefined =>
    pot.getOrElse(
      pot.map(profile, p => `${p.name} ${p.family_name}`),
      undefined
    )
);
```